src/login/logind-button.h

```diff
@@ -5,6 +5,16 @@ typedef struct Button Button;
 
 #include "logind.h"
 
+typedef enum
+{
```

Comment:
please follow our usual coding style, i.e. no line break before {. Also name both the enum and the typedef the same way.